src/pages/questionPage.js

```diff
@@ -23,13 +23,27 @@ export const initQuestionPage = (userInterface) => {
     .addEventListener('click', nextQuestion)
 };
 // check answers if correct or not 
-const getAnswer = (e)=> {
+let isSelectedOneAnswer = false;
+
+const getAnswer = (e) => {
   const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+if(!isSelectedOneAnswer){
   const answer = e.target.innerText;
-  currentQuestion.selected=answer.charAt(0)
-  currentQuestion.selected===currentQuestion.correct ? console.log('selected correct answer') : console.log('selected wrong answer')
+  currentQuestion.selected = answer.charAt(0)
+  const isAnswerCorrect = currentQuestion.selected===currentQuestion.correct
+  showAnswerIsCorrect(isAnswerCorrect,e.target)
+  isSelectedOneAnswer = true;
+}
 }
+// Show user if user selection correct or not
+const showAnswerIsCorrect = (isAnswerCorrect,target) => {
+  isAnswerCorrect ? setBackgroundColor('green',target) : setBackgroundColor('red',target)
+  }
```

Comment:
Can you please make sure the indentations are correct? In this line, for example, there should be no indentation. 

L 42,43 and 44 is also not correct.

You are also missing `;` in many lines, like 32,33, 34,40. Please, make sure they all have it.

For the rest I think it's good!!

Reply:
I think my prettier is not working for this project. I corrected them manually and now sending it.